docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -25,38 +23,16 @@ data:
       namespace: knative-eventing
 ```
 
-## Channel implementation options
+In this case, each new Broker created in the cluster would use by default the `MTChannelBasedBroker` broker class and the `config-br-default-channel` ConfigMap from the `knative-eventing` namespace for its configuration if not other specified in the brokers `eventing.knative.dev/broker.class` annotation and/or `.spec.config` (see [Developer configuration options](./broker-developer-config-options.md)).
 
-The following example shows a Broker object where the `spec.config` configuration is specified in a `config-br-default-channel` ConfigMap:
-
-```yaml
-apiVersion: eventing.knative.dev/v1
-kind: Broker
-metadata:
-  annotations:
-    eventing.knative.dev/broker.class: MTChannelBasedBroker
-  name: default
-spec:
-  # Configuration specific to this broker.
-  config:
-    apiVersion: v1
-    kind: ConfigMap
-    name: config-br-default-channel
-    namespace: knative-eventing
-```
```

Comment:
I believe showing the resulting resource would clarify the text, why removing the example?

Reply:
I removed it from the admin docs, because configuring it on the Broker object itself, can be done from developers. Currently we have something similar in the MTCCB Readme. Anyhow: IMO this would be better placed under the "Developer configuration options", but I didn't want to blow up this PR to much...

Reply:
If you're removing content please make sure it's moved to the developer section in this PR. Otherwise we're risking removing it in this PR and never adding it back in.

Reply:
Alternatively you can open a new PR to move this content